src/ControlComponents/DynamicPositioningMode.jsx

```diff
@@ -108,7 +113,10 @@ export default function DynamicPositioningMode({
   // Updates input-fields and the global DP settings to the current position
   const setCurrentPosition = () => {
     attributes.forEach(attribute => {
-      const currentPosition = Number(fromROV[attribute]);
+      let currentPosition;
+      attribute === 'yaw'
+        ? (currentPosition = radiansToDegrees(Number(fromROV[attribute])))
+        : (currentPosition = Number(fromROV[attribute]));
```

Comment:
simplify to
const currentPosition = (attribute === 'yaw') ? radiansToDegrees(Number(fromROV[attribute])) : Number(fromROV[attribute])

Reply:
🙌